api/.eslintrc.json

```diff
@@ -19,6 +19,7 @@
   "rules": {
     "@typescript-eslint/no-empty-function": "off",
     "@typescript-eslint/explicit-function-return-type": "warn",
+    "eqeqeq": "error",
```

Comment:
Something to consider might be to set this to "warn" instead of "error"

Either one will fail the `yarn lint` command and thus will fail in CircleCI (which is what we want).

The difference is that depending on your personal IDE settings, an eslint "error" will highlight red as you're writing the code, whereas an eslint "warn" will not "scream at you" in the IDE. It'll likely show up as a yellow suggestion instead of a red error. 

Either way, everyone would have to fix before pushing their code, which is good. But the warn/error would just change how it displays in the IDE before it's fixed. Personal preference.